sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -293,7 +290,15 @@ export interface Subscription {
 }
 
 // @public
-export interface SubscriptionOptions extends OptionalEventHandlers, EventProcessorOptions {
+export interface SubscriptionEventHandlers {
+    processClose?: ProcessCloseHandler;
+    processError?: ProcessErrorHandler;
+    processInitialize?: ProcessInitializeHandler;
+    processReceivedEvents: ProcessReceivedEvents;
```

Comment:
Did we decide against just `processEvents`? I think it should be clear that we're processing events that we're receiving, but can't remember if we decided to use this name specifically.

Reply:
I don't think we talked about that but I like your name better. I'll swap it.